dbms/src/Flash/Executor/QueryExecutor.h

```diff
@@ -58,7 +58,7 @@ class QueryExecutor
     virtual String getExtraJsonInfo() const { return "{}"; }
 
 protected:
-    virtual ExecutionResult execute(ResultHandler &&) = 0;
+    virtual ExecutionResult execute(ResultHandler &&, UInt64 minTSO_wait_time_in_ns) = 0;
```

Comment:
why not fill minTso_wait_time in `mpp_task_statistics.genExecutionSummaryResponse()`, seems hack here.

Reply:
Yes, it's a little track here to pass minTSO time as argument. The reason I don't handle it in mpp_task_statistics.genExecutionSummaryResponse() is: I want to explicitly show the minTSO time in source operator only, it is easier to do this in PipelineExec.

Reply:
Seems add a function like `ExecutorStatisticsCollector::addScheduleTimeForSource` is ok? And then the schedule time can also be displayed in the non-pipeline summary

Reply:
I see, I'll change it.

Reply:
Updated now.